extension/llm/export/builder.py

```diff
@@ -162,9 +164,18 @@ def capture_pre_autograd_graph(self) -> "LLMEdgeManager":
         # 2. torch.no_grad() is for getting rid of the dropout (not sure why training ops will show up)
         with torch.nn.attention.sdpa_kernel([SDPBackend.MATH]), torch.no_grad():
             # pyre-fixme[8]
-            self.pre_autograd_graph_module = capture_pre_autograd_graph(
-                self.model, self.example_inputs, dynamic_shapes=dynamic_shape
-            )
+            if self.args.qnn:
```

Comment:
nit: do we need to pass entire set of args? Can we not just pass in backend information? or `is_qnn`?

Reply:
So I'm not sure it's qnn specific and I feel like maybe coreml/mps might run into similar issue if they use pt2e flow....

Reply:
The current backend information is `args.qnn`, `args.coreml` and `args.mps`...We can refactor it to make it cleaner.

Reply:
> So I'm not sure it's qnn specific and I feel like maybe coreml/mps might run into similar issue if they use pt2e flow....

Yes... agreed on this 😮 
Is there any other backend trying the pt2e flow? (i.e., going through `convert_pt2e()`)